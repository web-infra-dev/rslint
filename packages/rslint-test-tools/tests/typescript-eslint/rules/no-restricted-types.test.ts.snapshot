exports[`no-restricted-types > invalid 1`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`bigint\` as a type. Use Ok instead.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 18
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`no-restricted-types > invalid 2`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`boolean\` as a type. Use Ok instead.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 19
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`no-restricted-types > invalid 3`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`never\` as a type. Use Ok instead.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 17
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`no-restricted-types > invalid 4`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`null\` as a type. Use Ok instead.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 16
        }
      }
    }
  ],
  "errorCount": 1,
  "fileCount": 7,
  "ruleCount": 1
}
`;

exports[`no-restricted-types > invalid 5`] = `
{
  "diagnostics": [
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`number\` as a type. Use Ok instead.",
      "filePath": "src/consistent-type-exports/type-only-exports.ts",
      "range": {
        "start": {
          "line": 6,
          "column": 21
        },
        "end": {
          "line": 6,
          "column": 27
        }
      }
    },
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`number\` as a type. Use Ok instead.",
      "filePath": "src/consistent-type-exports/type-only-exports.ts",
      "range": {
        "start": {
          "line": 12,
          "column": 10
        },
        "end": {
          "line": 12,
          "column": 16
        }
      }
    },
    {
      "ruleName": "no-restricted-types",
      "messageId": "bannedTypeMessage",
      "message": "Don't use \`number\` as a type. Use Ok instead.",
      "filePath": "src/virtual.ts",
      "range": {
        "start": {
          "line": 1,
          "column": 12
        },
        "end": {
          "line": 1,
          "column": 18
        }
      }
    }
  ],
  "errorCount": 3,
  "fileCount": 7,
  "ruleCount": 1
}
`;
