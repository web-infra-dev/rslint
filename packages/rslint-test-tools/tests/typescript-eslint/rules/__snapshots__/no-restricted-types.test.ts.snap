// Rstest Snapshot v1

exports[`no-restricted-types > invalid 1`] = `
{
  "diagnostics": [],
  "errorCount": 0,
  "fileCount": 73,
  "ruleCount": 1,
}
`;
